distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/NsiliClient.java

```diff
@@ -386,14 +388,30 @@ public void retrieveProductFromDAG(DAG dag) throws MalformedURLException {
                     + "\n");
 
             OrderRequest orderRequest = orderMgr.order(order, properties);
-
-            System.out.println("Completing OrderRequest...");
             DeliveryManifestHolder deliveryManifestHolder = new DeliveryManifestHolder();
             orderRequest.set_user_info("Alliance");
             PackageElement[] elements = null;
             try {
                 orderRequest.complete(deliveryManifestHolder);
 
+                // TODO: 9/12/16 Verify this is the best spot for calling emailOrder
+                if (isEmailEnabled) {
+                    order = createEmailOrder(orb, product, supportedPackageSpecs);
+
+                    //Validating Order
+                    System.out.println("Validating Email Order...");
```

Comment:
FYI: There is a PR to change `System.out` to SLF4J logging in the SDK NSILI packages